specification/README.md

```diff
@@ -18,11 +21,11 @@ Run the full build:
 `mvn install`
 
 Locate the html files:
-- target/generated-docs/Platform_Spec.html
-- target/generated-docs/WebProfile.html
-- target/generated-docs/ManagedBeans.html
+- `target/generated-docs/Platform-spec-<version>.html`
+- `target/generated-docs/WebProfile-spec-<version>.html`
+- `target/generated-docs/ManagedBeans.html`
```

Comment:
It's not yet generated, but it's going to need "-spec-<version>".

Reply:
Thanks.  Done.